webportal/src/app/job/job-submit/job-submit.schema.js

```diff
@@ -98,7 +98,7 @@ const taskRoleSchema = {
       format: "textarea",
       propertyOrder: 1001,
       options: {
-        expand_height: true,
+        // expand_height: true,
```

Comment:
What is the purpose?

Reply:
if the option is set to true, the editor should expand textarea automatically, but sometimes it doesn't work, so I explicitly set the style in css file.